frontend/_data/segments.js

```diff
@@ -98,10 +100,52 @@ const allSegmentData = async () => {
       }
     })
   );
+
+  await addObservations(segmentData);
```

Comment:
question: How long does it take for this to run? If it takes a significant amount of time and we want to speed up builds, or if we're rate-limited by weather.gov, would it be worth adding this to the cache?

If it's worth it, we can wrap this code in the `checkOrGet()` function, similar to the code a few lines above.

Reply:
So the good thing is no rate limits with weather.gov and it's quick, but still impacts the build. I tried wrapping and am not having luck but will try to look into further.

Reply:
Don't worry about it, I'll take care of it after merging; the API is kinda nonintuitive